src/main/java/pl/poznan/put/cs/scenariochecker/model/Step.java

```diff
@@ -8,4 +8,18 @@
 public class Step {
     private String content;
     List<Step> subSteps;
+
+    public Step(String content, List<Step> subSteps) {
```

Comment:
constructor and getters are unnecessary since we are using Lombok which does the boilerplate code for us